java/common/common-task/src/main/java/sleeper/common/task/RunDataProcessingTasks.java

```diff
@@ -85,9 +124,16 @@ public interface TaskLauncher {
         void launchTasks(int numberOfTasksToCreate, BooleanSupplier checkAbort);
     }
 
-    public void run(QueueMessageCount.Client queueMessageCount) {
+    public void runCompaction(QueueMessageCount.Client queueMessageCount) {
+        run(queueMessageCount, instanceProperties.get(COMPACTION_JOB_QUEUE_URL));
+    }
+
+    public void runBulkExport(QueueMessageCount.Client queueMessageCount) {
+        run(queueMessageCount, instanceProperties.get(LEAF_PARTITION_BULK_EXPORT_QUEUE_URL));
+    }
```

Comment:
It looks like these new methods don't really do what they say they do. I think there needs to be one place that determines which type of task is being run. Right now most of it is in the constructor, and there's also the CompactionTaskType enum. These methods can disagree with what's set in the constructor.